### ISSUE TITLE
Can't detect Java

### ISSUE THREAD

**Issue by Sammers21** [github/linguist#4370]
## Preliminary Steps

Please confirm you have...
- [x] reviewed [How Linguist Works](https://github.com/github/linguist#how-linguist-works),
- [x] reviewed the [Troubleshooting](https://github.com/github/linguist#troubleshooting) docs,
- [x] considered implementing an [override](https://github.com/github/linguist#overrides),
- [x] verified an issue has not already been logged for your issue ([linguist issues](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+repo%3Agithub/linguist)).

## Problem Description
Java files don't cosidered

### URL of the affected repository:
https://github.com/artificial-vulnerability-tools/env

### Last modified on:
2019-01-05

### Expected language:
Java

### Detected language:
Shell

**Comment by lildude** [comment]
This is happening because `env` is considered a vendored directory by Linguist:

https://github.com/github/linguist/blob/e761f9b013e5b61161481fcb898b59721ee40e3d/lib/linguist/vendor.yml#L228

... and all your Java files are in a directory called `env` hence they're ignored.

You'll need to implement an override to override this behaviour as I've done in a fork of your repo at https://github.com/lildude/env (I'll delete this at the end of the day on Wednesday).

Note, I had to make another change to a file (I added "Foo" to the README) to force a reanalysis as adding/changing the `.gitattributes` is not enough to kick off re-analysis.
